<?xml version="1.0"?>
<launch>
    <arg name="action_name" default="perceive_plane" />
    <arg name="server_name" default="/mdr_actions/perceive_plane_server" />
    <arg name="action_timeout" default="120" />
    <arg name="clear_plane_memory" default="true" />
    <arg name="action_dispatch_topic" default="/kcl_rosplan/action_dispatch"/>
    <arg name="action_feedback_topic" default="/kcl_rosplan/action_feedback"/>

    <node pkg="mdr_perceive_plane_action" type="perceive_plane_client" name="perceive_plane_client" output="screen" ns="mdr_actions">
        <param name="action_name" type="str" value="$(arg action_name)" />
        <param name="server_name" type="str" value="$(arg server_name)" />
        <param name="action_timeout" type="double" value="$(arg action_timeout)" />
        <param name="clear_plane_memory" type="bool" value="$(arg clear_plane_memory)" />
        <remap from="action_dispatch_topic" to="$(arg action_dispatch_topic)" />
        <remap from="action_feedback_topic" to="$(arg action_feedback_topic)" />
    </node>
</launch>
